### DISCUSSION TITLE
[Feature Request] Changing the working directory path of Zed's integrated terminal

### DISCUSSION THREAD

**Discussion by ekaradon** [zed-industries/zed#14171] [Feature Requests]
It would be convenient for monorepo. Visual studio code allows it through: `terminal.integrated.cwd`. Could it be possible also for zed? See there: https://code.visualstudio.com/docs/terminal/basics#_working-directory

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
